level_elements/level.gd

```diff
@@ -362,21 +380,12 @@ func place_goal(coord: Vector2i) -> void:
 func place_tile(tile_coord: Vector2i) -> void:
 	if level_data.tiles.get(tile_coord): return
 	if is_space_occupied(Rect2i(tile_coord * 32, Vector2i(32, 32)), [&"tiles"]): return
-	level_data.tiles[tile_coord] = 1
+	level_data.tiles[tile_coord] = true
```

Comment:
Okay, then I'd do so everywhere else, I think it doesn't do that yet in the save load. Also in this case maybe use `level_data.tiles.has(tile_coord)` instead of `get(tile_coord)` to make sure it doesn't unnecessarily need to query the value and cast it